doc/backing-store-crd.md

```diff
@@ -16,13 +22,17 @@ Adding support for a new type of backing-store is rather easy as it requires jus
 - CR: [noobaa.io_v1alpha1_backingstore_cr.yaml](../deploy/crds/noobaa.io_v1alpha1_backingstore_cr.yaml)
 
 
-# Reconcile
+# Reconciliation
+It is possible to create backingstores by using the NooBaa CLI tool (from here on referred to in shell commands as `noobaa`), or by applying YAMLs.
 
-#### AWS-S3 type
+From here on out, fields that depend on the user's choice and input will be marked with `<>`. In cases where the value might not be inferrable from context, an additional explanation might be included. 
 
-Create a cloud resource within the NooBaa brain and use S3 API for storing encrypted chunks of data in the AWS cloud.
+If you choose to use the CLI's `--secret-name` option, or to apply a YAML, please see [Store Secret Creation](store-connection-secrets.md).
+
+## AWS S3
+Uses the S3 API for storing encrypted chunks of data in AWS buckets
 ```shell
-noobaa -n noobaa backingstore create aws-s3 bs --access-key KEY --secret-key SECRET --target-bucket BUCKET
+noobaa -n noobaa backingstore create aws-s3 <BACKINGSTORE NAME> --access-key <> --secret-key <> --target-bucket <>
```

Comment:
Why use capital letters?
For example, in Git [documentation](https://git-scm.com/docs/git-checkout): `git checkout [<branch>]`.

And according to your style in the file, it should be only `<>`.
In my opinion, using the brackets with a short description is more clear than just `<>`.
Using the example above `git checkout <branch>` is clearer than `git checkout <>.`